### PR TITLE
fix361Coverage

### PR DESCRIPTION
**Pull request by RalphSteinhagen**
* fixed System.err.println(..) left-over from previous commit
* updated AbstractAxisParameter unit-tests,
* added some AbstractAxis unit-tests

### CODE REVIEW

File: chartfx-chart/src/test/java/de/gsi/chart/axes/spi/AbstractAxisParameterTests.java [hunk 1]
```diff
@@ -55,11 +57,19 @@ public void testAutoGetterSetters() {
...
+        axis.setAutoGrowRanging(true);
+        assertTrue(axis.isAutoGrowRanging());
+        axis.setAutoRanging(true);
+        assertTrue(axis.isAutoRanging());
+        assertFalse(axis.isAutoGrowRanging());
     }
 
     @Test
-    public void testBasicGetterSetters() {
+    void testBasicGetterSetters() {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/axes/spi/AbstractAxisParameterTests.java:70]: Codacy found an issue: [Avoid really long methods.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6223548)

**Comment by RalphSteinhagen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           

Complexity increasing per file
==============================
- chartfx-chart/src/test/java/de/gsi/chart/axes/spi/AbstractAxisTests.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6223548&bid=20284977)

File: chartfx-chart/src/test/java/de/gsi/chart/axes/spi/AbstractAxisTests.java [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+import de.gsi.chart.ui.geometry.Side;
+import de.gsi.chart.ui.utils.JavaFXInterceptorUtils;
+import de.gsi.chart.utils.FXUtils;
+
+@ExtendWith(ApplicationExtension.class)
+@ExtendWith(JavaFXInterceptorUtils.SelectiveJavaFxInterceptor.class)
+class AbstractAxisTests {
+    private static final int WIDTH = 300;
+    private static final int HEIGHT = 200;
+    private Pane pane;
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/axes/spi/AbstractAxisTests.java:42]: Codacy found an issue: [Perhaps 'pane' could be replaced by a local variable.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6223548)

**Review by wirew0rm — APPROVED** [review]
Looks good!

**Pull request merged**